enterprise/frontend/src/embedding-sdk/components/public/InteractiveQuestion/components/BackButton.tsx

```diff
@@ -0,0 +1,27 @@
+import { t } from "ttag";
+
+import { useInteractiveQuestionContext } from "embedding-sdk/components/public/InteractiveQuestion/context";
+import { Button, Icon, Tooltip } from "metabase/ui";
+
+const sizeOverrideStyles = {
+  width: 32,
+  height: 32,
+};
+
+export const QuestionBackButton = () => {
+  const { onNavigateBack } = useInteractiveQuestionContext();
```

Comment:
I wonder if bringing the entire context to use only a specific subset of things will affect number of re-rendering needed. e.g. if something unrelated to `onNavigateBack` changes, the QuestionBackButton will re-render.